ansible_base/resource_registry/models/resource.py

```diff
@@ -40,28 +42,28 @@ def get_resource_config(self):
 
 
 class Resource(models.Model):
-    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE, related_name="resources")
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE, related_name="resources", help_text=_("The content type for this resource."))
 
     # this has to accommodate integer and UUID object IDs
-    object_id = models.TextField(null=False, db_index=True)
+    object_id = models.TextField(null=False, db_index=True, help_text=_("The object id for this resource."))
     content_object = GenericForeignKey('content_type', 'object_id')
 
     service_id = models.UUIDField(
         null=False,
         default=service_id,
-        help_text="ID of the service responsible for managing this resource.",
+        help_text=_("ID of the service responsible for managing this resource."),
     )
 
     # we're not using this as the primary key because the ansible_id can change if the object is
     # externally managed.
-    ansible_id = models.UUIDField(default=uuid.uuid4, db_index=True, unique=True)
+    ansible_id = models.UUIDField(default=uuid.uuid4, db_index=True, unique=True, help_text=_("A unique ID identifying this resource in AAP."))
 
     # human readable name for the resource
-    name = models.CharField(max_length=512, null=True)
+    name = models.CharField(max_length=512, null=True, help_text=_("The name of this resource."))
```

Comment:
It might not be the name...

Remind me again why we have this? Oof, ask @newswangerd

Reply:
@AlanCoding to make it human readable/easier to search. You're right, it can be anything identifiable. It's the username for users.